modules/network-observability-pktdrop-overview.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-pktdrop-overview_{context}"]
+= Configuring packet drop tracking
+You can configure graphical representation of network flow records with packet loss in the *Overview* view. By employing eBPF tracepoint hooks, you can gain valuable insights into packet drops for TCP, UDP, SCTP, ICMPv4, and ICMPv6 protocols, which can result in the following actions:
```

Comment:
there should be mention of OVS drops too right @msherif1234 now that it is implemented

Reply:
I asked @msherif1234 and he said ovs drops require a kernel patch which won't be available when OCP 4.14 ship but it may go in a z-stream. So I guess, we don't need to mention OVS in the docs for 1.4 timeframe but maybe slightly post. Thanks for mentioning this @Amoghrd ; its good to have more awareness of the feature.

Reply:
yup got it okay